test/api_application/api_application_test.rb

```diff
@@ -37,23 +37,9 @@ def test_api_middleware_stack
   private
 
   def expected_middleware_stack_rails3
-    [
-      "ActionDispatch::Static",
-      "Rack::Lock",
-      "ActiveSupport::Cache::Strategy::LocalCache",
-      "Rack::Runtime",
-      "ActionDispatch::RequestId",
-      "Rails::Rack::Logger",
-      "ActionDispatch::ShowExceptions",
-      "ActionDispatch::DebugExceptions",
-      "ActionDispatch::RemoteIp",
-      "ActionDispatch::Reloader",
-      "ActionDispatch::Callbacks",
-      "ActionDispatch::ParamsParser",
-      "ActionDispatch::Head",
-      "Rack::ConditionalGet",
-      "Rack::ETag"
-    ]
+    expected_middleware_stack_rails.map do |x|
+      x == "ActionDispatch::Head" ? "Rack::Head" : x
```

Comment:
@spastorino shouldn't be the other way around? I feel it like this given the diff:

```ruby
x == "Rack::Head" ?  "ActionDispatch::Head" : x
```